traffic_portal/app/src/modules/private/configure/deliveryServices/urlSigKeys/DeliveryServiceUrlSigKeysController.js

```diff
@@ -0,0 +1,100 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+var DeliveryServiceUrlSigKeysController = function(deliveryService, urlSigKeys, $scope, $state, locationUtils, deliveryServiceService, deliveryServiceUrlSigKeysService, $uibModal) {
+	$scope.deliveryService = deliveryService;
+	$scope.urlSigKeys = Object.keys(urlSigKeys).map(function(key) {
+			return {sortBy: parseInt(key.slice(3)), label: key, urlSigKey: urlSigKeys[key]};
```

Comment:
maybe a comment here about what exactly is going on here...or maybe an example of what the result is?